index.js

```diff
@@ -13,6 +13,9 @@ module.exports = {
       case 'build':
         require('./cli/build')(argv, skyPagesConfig, webpack);
         break;
+      case 'build-public-library':
+        require('./cli/build-public-library')(skyPagesConfig);
```

Comment:
I probably originally did it for testing, but should we pass in webpack here to stay consistent?